src/bluehawk/parser/lexer/makeLexer.ts

```diff
@@ -38,3 +37,16 @@ export function makeLexer(languageTokens: TokenType[]): Lexer {
     defaultMode: "RootMode",
   });
 }
+
+// Helper function for generating command attribute pop tokens
+function _CommandAttributesPopTokens(languageTokens: TokenType[]): TokenType[] {
+  let popTokens: TokenType[] = [{ ...Newline, POP_MODE: true }];
```

Comment:
popTokens is not reassigned. (JS 'const' doesn't really imply immutability of value, just of reference)
```suggestion
  const popTokens: TokenType[] = [{ ...Newline, POP_MODE: true }];
```

Reply:
This decision was informed by [this passage from _You Don't Know JS_](https://github.com/getify/You-Dont-Know-JS/blob/2nd-ed/get-started/ch2.md):

> It's ill-advised to use const with object values, because those values can still be changed even though the variable can't be re-assigned. This leads to potential confusion down the line...

Open to further discussion on this, but I think Kyle Simpson does not support using const for object types like arrays and objects. 

I'll make the change as I don't have a real preference either way on this, just thought I'd give context for why this was done.

Reply:
Our eslint rules _should_ change this automatically, so no point fighting it.

I also think it's a stretch to arbitrarily add a semantic meaning to `const`/`let` that's not actually supported in the language.